### PR TITLE
small chat dark mode fixes

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] chat header description on desktop/ipad
- [x] fs upload footer

### CODE REVIEW

File: shared/chat/header.tsx [hunk 1]
```diff
@@ -31,36 +31,40 @@ type Props = {
...
+
+  const descStyleOverride = React.useMemo(() => {
+    const descStyleMobile = {
+      color: Styles.globalColors.black_50,
+      fontSize: 13,
+      lineHeight: 17,
+    }
+    const descStyleDesktop = {
+      fontSize: 13,
+      lineHeight: 17,
```
Comment (buoyad) [inline shared/chat/header.tsx]: this needs `px`

Comment (chrisnojima) [reply shared/chat/header.tsx]: ok then  we need ti fix it further in markdown cause px isn't allowed in teh style and we convert it elsewhere

Comment (chrisnojima) [reply shared/chat/header.tsx]: updated

File: shared/common-adapters/markdown/index.desktop.tsx [hunk 2]
```diff
@@ -1 +1,29 @@
...
+import {SimpleMarkdownComponent} from './shared'
+import {Props} from '.'
+
+// We convert lint heights to px here
+const DesktopSimpleMarkdownComponent = (props: Props) => {
+  const {styleOverride, ...rest} = props
+
+  const fixedStyleOverride = React.useMemo(
+    () =>
+      styleOverride
```
Comment (buoyad) [inline shared/common-adapters/markdown/index.desktop.tsx]: There's `unifyStyles` (that `platformStyles` uses for this) in `styles/shared`, looks like you could use that

**Comment by chrisnojima** [comment]
Maybe I'll switch it to use platform styles instead

**Comment by chrisnojima** [comment]
done, much cleaner tnx

**Pull request merged**